pkg/model/dev.go

```diff
@@ -430,6 +437,7 @@ func (dev *Dev) GevSandbox() *appsv1.Deployment {
 					Labels: map[string]string{
 						"app": dev.Name,
 					},
+					Annotations: dev.Annotations,
```

Comment:
This code is only executed when `okteto up` creates a new deployment for scratch.
If you also want to add the annotations when `okteto up` modifies an existing deployment, you should assign the annotations in the `translate` logic, somewhere over here:
https://github.com/okteto/okteto/blob/master/pkg/k8s/deployments/translate.go#L73

Reply:
Thanks for pointing that out.
I wasn't too sure where to put it, but now it makes sense.